app/components/QuestionCard/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ *
+ * QuestionCard
+ *
+ */
+
+import { createStyleSheet } from 'jss-theme-reactor';
+import Avatar from 'material-ui/Avatar';
+import Card, { CardContent } from 'material-ui/Card';
+import Divider from 'material-ui/Divider';
+import List, { ListItem } from 'material-ui/List';
+import { grey } from 'material-ui/styles/colors';
+import Typography from 'material-ui/Typography';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import moment from 'moment';
+import React, { PropTypes } from 'react';
+import Markdown from '../Markdown';
+
+const styleSheet = createStyleSheet('SimpleCard', {
+  avatar: {
+    marginRight: 16,
+    flexShrink: 0,
+  },
+  avatarH: {
+    extend: 'avatar',
+    marginTop: 8,
+    transform: 'scale(1.5)',
+  },
+  lItem: {
+    alignItems: 'flex-start',
+    paddingTop: 12,
+    paddingRight: 16,
+    paddingBottom: 12,
+    paddingLeft: 16,
+    backgroundColor: grey[50],
+
+  },
+  lItemH: {
+    extend: 'lItem',
+    backgroundColor: grey[200],
+  },
+  content: {
+    padding: 0,
+    '&:last-child': {
+      paddingBottom: 0,
+    },
+  },
+});
+
+const image = require('./korri.jpg');
+const localeSpec = {
```

Comment:
@marcmos nie mam pojęcia, jest chyba coś takiego jak language provider ale nie wiem jak to dokładnie działa